dspace-spring-rest/src/test/java/org/dspace/app/rest/builder/WorkspaceItemBuilder.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.builder;
+
+import org.dspace.content.Collection;
+import org.dspace.content.Item;
+import org.dspace.content.WorkspaceItem;
+import org.dspace.content.service.DSpaceObjectService;
+import org.dspace.core.Context;
+import org.dspace.eperson.Group;
```

Comment:
this is not used and lead to the checkstyle violation reported by travis, see https://travis-ci.org/DSpace/DSpace/builds/399324480#L5632